src/main/java/com/zipcodewilmington/StringArrayUtils.java

```diff
@@ -42,31 +43,45 @@ public static String getSecondToLastElement(String[] array) {
      * @return true if the array contains the specified `value`
      */ // TODO
     public static boolean contains(String[] array, String value) {
-        return false;
+        return Arrays.asList(array).contains(value);
     }
 
     /**
      * @param array of String objects
      * @return an array with identical contents in reverse order
      */ // TODO
     public static String[] reverse(String[] array) {
-        return null;
+        String[] reversed = new String[array.length];
+        for(int i = 0; i < array.length; i++) {
+            reversed[array.length - 1 - i] = array[i];
+        }
+        return reversed;
     }
 
     /**
      * @param array array of String objects
      * @return true if the order of the array is the same backwards and forwards
      */ // TODO
     public static boolean isPalindromic(String[] array) {
-        return false;
+        return Arrays.toString(array).equals(Arrays.toString(reverse(array)));
```

Comment:
`return Arrays.equals(array, reverse(array))`

Reply:
Oh nice! that's a lot simpler. Didn't know arrays had that specific function